source/dub/init.d

```diff
@@ -90,6 +90,15 @@ void initPackage(Path root_path, string[string] deps, string type,
 private void initMinimalPackage(Path root_path, ref PackageRecipe p)
 {
 	p.description = "A minimal D application.";
+
+	// we want to create a empty, not-null dictionary,
+	// s. t. the generated recipe contains an empty "dependencies" section
+	// in case of PackageFormat.json
+	auto bar = Dependency("~>0.0.0");
+	p.buildSettings.dependencies["foo"] = bar;
+	p.buildSettings.dependencies.remove("foo");
```

Comment:
Would it make more sense to change the json writer to always add a dependencies section?